pkgs/os-specific/linux/sysdig/default.nix

```diff
@@ -72,6 +71,13 @@ stdenv.mkDerivation rec {
 
     cp -r ${driver} driver-src
     chmod -R +w driver-src
+
+    # Hacky but needed until https://github.com/draios/sysdig/issues/2077 is resolved for kernel >= 6.8 as strlcpy got removed and build fails
+    ${lib.optionalString
+    (kernel != null && lib.versionAtLeast kernel.version "6.8") ''
```

Comment:
> I'd guess it's safe to do in all kernel versions we support.
I wouldn't do this it's only removed from kernel >= 6.8 in the kernel-sources build against and so only causing an issue there as the function definition is missing even it would never be used because it has a check in the pm_events.c for kernel-version.